src/components/Favorite.jsx

```diff
@@ -1,56 +1,44 @@
-import React, { Component } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
```

Comment:
nitpick: Since we're following `deriv-app` code conventions to , can just keep the import as `React` and we use `React.useCallback`, `React.useEffect` and so on instead?